app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt

```diff
@@ -56,7 +56,7 @@ class AdministratorControlsFragmentPresenter @Inject constructor(
           /* attachToRoot= */ false
         )
 
-    internalProfileId = activity.intent.getIntExtra(NAVIGATION_PROFILE_ID_ARGUMENT_KEY, -1)
+    internalProfileId = activity.intent.extractCurrentUserProfileId().internalId ?: -1
```

Comment:
```suggestion
    val internalProfileId = activity.intent.extractCurrentUserProfileId().internalId
```
`CurrentUserProfileIdIntentDecorator.extractCurrentUserProfileId()` returns a non-null value, since you added a non-null check before adding the profileId.

Reply:
Right.